recipes/scikit-posthocs/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and pasting from another recipe, but not really needed.
+
+package:
+  name: scikit-posthocs
+  version: "0.4.0"
+
+source:
+  url: https://github.com/huidongchen/scikit-posthocs/archive/master.zip
+  sha256: b29db2003ba59ca9422dee2cb43387bb1c395a04766a96f0b7b791e879c8071f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python
+  host:
+    - python
+    - setuptools
+    - scipy
+    - numpy
+    - statsmodels
+    - pandas>=0.20.0
+    - matplotlib
+    - seaborn
+    - pip
+
+  run:
+    - python
+    - scipy
+    - numpy
```

Comment:
This line needs to be a jinja2 pin compatible expression. Google for the conda forge docs and the proper thing is listed there.